projects/asthma-copd/nbs/05_gwas/py/05-gwas-inflation_factor.py

```diff
@@ -0,0 +1,194 @@
+# ---
+# jupyter:
```

Comment:
I might just be missing something, but is this supposed to be in `.py` format? It looks like it might be `.ipynb` to me.

Reply:
That's a good question. Here I'm using `jupytext` to sync a Jupyter Notebook (`.ipynb`) with a Python code file (`.py`). This makes diffs better, improving review for instance. So for each Jupyter Notebook, I have two files, the `.ipynb` and the `.py`. Both are automatically synced.